decentraland/redux/actions.js

```diff
@@ -18,6 +20,13 @@ export function setAccountCreationStatus(creationStatus) {
   return { type: SET_ACCOUNT_CREATION_STATUS, creationStatus };
 }
 
+export function updateAccountCreationCurrentStatusAction(creationAction) {
```

Comment:
I can't tell from the naming if this is supposed to update the list with a new item or update a single item in the list. Or both.

https://github.com/tasitlabs/tasit/pull/260#discussion_r273046928

Reply:
`updateActionOfAccountCreationCurrentStatus` sounds better?

Reply:
Hmm, that's kind of hard to parse too. It's the word `update` that is confusing me. Is this action for adding a new item to a list or changing a single item in the list that already exists? Or something else?

Reply:
The intent is to add an action but if the current status an action already has an action, it will be updated.

Reply:
Understood. In that case, I like the use of the word `update` like with a REST API after all. The only thing I think we should change is removing the word `Current`

Reply:
I made these changes. Feel free to resolve this if you're okay with the new name.

Reply:
Ok, resolving this.